### PR TITLE
feat: Micro resource

### PR DESCRIPTION
**Pull request by yoshwata**
### Objective
Expantion of https://github.com/screwdriver-cd/executor-k8s/pull/79

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -115,8 +115,10 @@ class K8sVMExecutor extends Executor {
      * @param  {Number} [options.kubernetes.buildTimeout=90]          Number of minutes to allow a build to run before considering it is timed out
      * @param  {String} [options.kubernetes.resources.cpu.high=6]     Value for HIGH CPU (in cores)
      * @param  {Number} [options.kubernetes.resources.cpu.low=2]      Value for LOW CPU (in cores)
+     * @param  {Number} [options.kubernetes.resources.cpu.micro=0.5]  Value for LOW CPU (in cores)
```
Comment (minz1027) [inline index.js]: Should fix the comment Value for MICRO CPU (in cores)

Comment (yoshwata) [reply index.js]: Thank you. I've fixed.

**Comment by tk3fftk** [comment]
I think the `MICRO` default CPU value of k8s and k8s-vm should be same value. The difference makes it a little confusing. How about this?

**Comment by yoshwata** [comment]
@tkyi Thank you! I resolved conflicts.

**Pull request merged**